### PR TITLE
Lesson9

### PR DESCRIPTION
**Pull request by ameyunaikgithub**

### CODE REVIEW

File: webcam/public/index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    Name:<input type="text" id="Name" />
+    Comment:<input type="text" id="Comment" />
+  <!--<input type="submit" id="button" />-->
+    Picture:<input type="submit" value="take picture" id = "click"/>
+      <pre id="chatid"/>
+      <video id = "v1"></video>
+      <canvas width = 400 height = 400 id ='c1'></canvas>
+
+ <script>
+/*const button =document.getElementById("button");
```
Comment (allopez7) [inline webcam/public/index.html]: Get rid of quotes.

**Review by allopez7 — APPROVED** [review]
Good job!

File: meme/meme.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+/*response of button 2*/
+app.post('/picture',(req,res)=>{
+  let picData = req.body.data.replace('data:image/png;base64');
+  let picPath = `/home/amey/garageScript/meme/public/pics/${req.body.name}.png`;
+  let memePath = `/home/amey/garageScript/meme/public/meme/${req.body.name}.png`;
+  /* we call it an empty function inside a function since we are dealing with image files and allowing some time to get the image  uploaded*/
+  fs.writeFile(picPath,picData, 'base64' ,() => {
+    gm(picPath).fontSize(40).drawText(50,50,req.body.comment).write(memePath, ()=>{});
+});
```
Comment (songz) [inline meme/meme.js:27]: indent is wrong.

File: meme/public/index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+}).then((mediaStream) =>{
+  video.srcObject =mediaStream;
+  video.play();
+}).catch((err) => {
+  console.log("err");
+});
+//* taking a picture *//
+const button2 =document.getElementById("click");
+button2.onclick=()=>{
+  const name =document.getElementById("Name").value;
```
Comment (songz) [inline meme/public/index.html:55]: you should define variables right before you use them. Therefore, move line 55+56 to line 62

**Review by songz — APPROVED** [review]

File: meme/meme.js (hunk 2 above)
Comment (ameyunaikgithub) [reply meme/meme.js:27]: hey song i changed it.
thanks

**Pull request merged**